backend/server/models/questionnaire-response.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+module.exports = (sequelize, DataTypes) => {
+  var QuestionnaireResponse = sequelize.define('QuestionnaireResponse', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    // In future, when Participant model is created, there will be a foreign key to it.
+  }, {});
+
+  QuestionnaireResponse.associate = function (models) {
+    QuestionnaireResponse.belongsTo(models.Questionnaire, {
```

Comment:
Don't we need a foreign key to Experiment? Each questionnaire belongs to a user, but we somehow need to link some of its responses to both a specific participant and specific experiment, right?

Reply:
You are correct; this will be required. However, this will be achieved by using the foreign key of the Participant model (which corresponds to an individual who participates in an experiment), and will therefore be able to link indirectly a QuestionnaireResponse with an Experiment (QuestionnaireResponse has foreign key to Participant; Participant has foreign key to Experiment).